kumascript/macros/Compat.ejs

```diff
@@ -30,13 +30,19 @@ Example calls
 
 */
 
-var query = $0 || env['browser-compat'];
+// Read 'browser-compat' key
+const query = $0 || env['browser-compat'];
 if (!query) {
   throw new Error("No first query argument or 'browser-compat' front-matter value passed");
 }
-var depth = $1 || 1;
-var queries = Array.isArray(query) ? query : [query];
-var output = queries.map(query => `<div class="bc-data" data-query="${query}" data-depth="${depth}" data-multiple="${queries.length > 1}">
+
+// Read 'polyfills' key
+const polyfills = env['polyfills'] || "";
+
+const depth = $1 || 1;
+const queries = Array.isArray(query) ? query : [query];
+
+var output = queries.map(query => `<div class="bc-data" data-query="${query}" data-depth="${depth}" data-multiple="${queries.length > 1}" data-polyfills="${polyfills}">
```

Comment:
Hm, this would mean that the same polyfills would be printed multiple times for each query. Not ideal, but that almost makes me think that we need a single separate `<div class="bc-polyfills">` element instead of adding this to this `div.bc-data` element.